### PR TITLE
[PHPUnit bridge] Bump php version of PHPUnit-bridge

### PR DESCRIPTION
**Pull request by gmponos**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no (?)
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

Related to https://github.com/symfony/symfony/pull/29623#issuecomment-450400625

I checked the BC page I think I couldn't find any related to BC changes... anyway...

As far as I understood from the comment above I must target `master` branch.

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/bin/simple-phpunit [hunk 1]
```diff
@@ -54,8 +54,10 @@ if (PHP_VERSION_ID >= 70100) {
 } elseif (PHP_VERSION_ID >= 50600) {
     // PHPUnit 5 requires PHP 5.6+
     $PHPUNIT_VERSION = $getEnvVar('SYMFONY_PHPUNIT_VERSION', '5.7');
-} else {
+} elseif (PHP_VERSION_ID >= 50500) {
     $PHPUNIT_VERSION = '4.8';
+} else {
+    throw new \Exception('Symfony phpunit-bridge does not support your current PHP version.');
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit]: no need for that part - composer does the job

**Review by nicolas-grekas — COMMENTED** [review]
while bumping to phpunit5 & php5.5, there are many opportunities to improve the code (eg $this in closure, and polyfills that we could remove, see "Legacy" namespace.)
Up to do it?

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Bridge/PhpUnit/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * added `ClassExistsMock`
+ * bumped PHP version from 5.5.3 to 5.5.9
```
[the remark this answers is not in the document]
Comment (gmponos) [reply src/Symfony/Bridge/PhpUnit/CHANGELOG.md]: 🤦‍♂️... Yes it is a type.. fixed.

**Comment by greg0ire** [comment]
The `Legacy` namespace should stay around for a while, but some simplifications seem indeed possible: https://3v4l.org/MmG7J I'm working on a PR right now.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @gmponos.

**Pull request merged**